source/extensions/filters/network/redis_proxy/conn_pool_impl.cc

```diff
@@ -83,10 +84,11 @@ InstanceImpl::makeRequestToHost(const std::string& host_address,
   return tls_->getTyped<ThreadLocalPool>().makeRequestToHost(host_address, request, callbacks);
 }
 
-InstanceImpl::ThreadLocalPool::ThreadLocalPool(std::shared_ptr<InstanceImpl> parent,
-                                               Event::Dispatcher& dispatcher,
-                                               std::string cluster_name)
+InstanceImpl::ThreadLocalPool::ThreadLocalPool(
+    std::shared_ptr<InstanceImpl> parent, Event::Dispatcher& dispatcher, std::string cluster_name,
+    const Extensions::Common::DynamicForwardProxy::DnsCacheSharedPtr& dns_cache)
     : parent_(parent), dispatcher_(dispatcher), cluster_name_(std::move(cluster_name)),
+      dns_cache_(dns_cache), default_port_(static_cast<uint16_t>(6379)),
```

Comment:
Same comment about named constant for 6379, also the mem var can be constant I think. And if it's always constant and not configurable do we even need any of these member variables for now?

Reply:
yeah let's drop it as a member var for now and make it a constant somewhere accessible.